apps/api/infra/api.ts

```diff
@@ -132,6 +132,16 @@ export const serviceSetup = (services: {
       PARTY_LETTER_REGISTRY_BASE_API_URL: ref(
         (h) => `http://${h.svc(services.servicesPartyLetterRegistryApi)}`,
       ),
+      DRIVING_LICENSE_PATH_REPLACEMENT_FROM: {
+        prod: '/RafraentOkuskirteini-v1/api/Okuskirteini/',
+        staging: '',
```

Comment:
Use the constant `MissingSetting` to be more semantic

Reply:
ah ok, thanks

Reply:
After talking to @arnorhs turns out empty string is the intended values here.